Realm/RLMRealm.mm

```diff
@@ -460,8 +461,8 @@ + (void)resetRealmState {
 
 - (void)verifyNotificationsAreSupported {
     [self verifyThread];
-    if (_realm->config().read_only()) {
-        @throw RLMException(@"Read-only Realms do not change and do not have change notifications");
+    if (_realm->config().immutable()) {
+        @throw RLMException(@"Immutable Realms do not change and do not have change notifications");
```

Comment:
The term we use in this error message needs to match the term used in the Cocoa API, which remains read-only.